tests/phases/design/test_array_system_design.py

```diff
@@ -69,7 +69,7 @@ def test_cable_not_found():
 
 
 @pytest.mark.parametrize(
-    "config,num_full_strings,num_partial_strings,num_turbines_full_string,num_turbines_partial_string",
+    "config,num_full_strings,num_partial_strings,num_turbines_full_string,num_turbines_partial_string",  # noqa: E501
```

Comment:
noqa here. Is Noqa specifically for Flake8 or do other linters pick it up?

Reply:
E501 is the long line error. I probably forgot to return to this after running tests and all that to identify more of these.